### PR TITLE
Usernames with capital letters fail to authenticate

### PR DESCRIPTION
**Pull request by alterx**
Fixes https://github.com/hoodiehq/hoodie-account-server/issues/271 by storing the lowercase version of name and accountKey.

### CODE REVIEW

File: lib/accounts/add.js [hunk 1]
```diff
@@ -42,6 +57,7 @@ function addAccount (state, properties, options) {
   })
 
   .catch(function (error) {
+    console.log(error)
```
Comment (gr2m) [inline lib/accounts/add.js]: can you remove the console.log?

**Comment by alterx** [comment]
The changes you requested are ready @gr2m

**Comment by gr2m** [comment]
I’d like to invite you to the Hoodie Contributors Team, that way you don’t need to for the repos anymore and can directly create branches on the repositories.

Are you on twitter, we love giving new contributors a shout out :)

**Comment by alterx** [comment]
Of course, I'm working on an application and using hoodie for it so I'll probably be submitting issues and PRs every time I find something.

And yes, my handle is @__el_Negro

**Comment by LowProfileDog** [comment]
Welcome to the Hoodie Contributors Team 🐶

**Pull request merged**